netmanager/src/views/components/Sites/SiteView.js

```diff
@@ -53,6 +53,7 @@ const SiteForm = ({ site }) => {
   };
 
   const handleSubmit = async () => {
+    if(isEmpty(siteInfo.airqlouds)) { delete(siteInfo.airqlouds) } //empty airqloud array deleted before submit
```

Comment:
Thanks for this. @FaithDaka , great initiative!
Could we perhaps employ some library which removes all empty fields before submission?
Just to cater for future occurrences like this one? An example library is **clean-deep**: https://www.npmjs.com/package/clean-deep
**Context:**
I am thinking about a solution which can be able to address any future occurrences of a similar problem. So that we can completely erase this problem and prevent it from ever happening again. :)

Reply:
Yes, I'd thought of that as well. Being able to drop empty fields dynamically would be great, though I wanted to first find where those fields come from in the first place. Because as per the code, only data from changed input fields is posted to the server so how do certain empty fields end up appended to the route as well?
Let me check out the library. Thank you!